test/project_types/script/ui/error_handler_test.rb

```diff
@@ -151,6 +151,13 @@ def should_call_display_and_raise
         end
       end
 
+      describe "when MetadataNotFoundError" do
+        let(:err) { Script::Layers::Domain::Errors::MetadataNotFoundError.new(filename: "filename") }
```

Comment:
This should not be a kwarg. It doesn't break at the moment because StandardError can take kwargs, but this doesn't line up with how the `MetadataNotFoundError` initializer expects.